tests/test_worker_palavras_raw.py

```diff
@@ -28,6 +29,7 @@
 
 class TestPalavrasRawWorker(TaskTest):
 
+    @skipIf(not palavras_raw.palavras_installed(), 'palavras software is not installed')
     def test_should_run_only_if_language_is_portuguese(self):
```

Comment:
I think I agree with you. Maybe we should either change this to test portuguese and english or change the name to something like "does not run palavras if document is in english".